create-rust-app/src/util/poem_utils.rs

```diff
@@ -127,15 +134,14 @@ pub async fn render_views(uri: &Uri) -> impl IntoResponse {
     template_response(uri, content)
 }
 
-fn template_response(uri: &Uri, content: String) -> Response {
+fn template_response(_uri: &Uri, content: String) -> Response {
+    #[cfg(not(debug_assertions))]
+    let content = content;
+    #[cfg(debug_assertions)]
     let mut content = content;
     #[cfg(debug_assertions)]
     {
-        let uri = Uri::from_str(req.connection_info().host());
-        let hostname = match &uri {
-            Ok(uri) => uri.host().unwrap_or("localhost"),
-            Err(_) => "localhost",
-        };
+        let hostname = _uri.host().unwrap_or("localhost");
         let inject: &str = &format!(
```

Comment:
This is the only someone significant change that isn't directly related to formatting or the CI checks is.